src/fulfiller.js

```diff
@@ -3,14 +3,14 @@
  */
 class Fulfiller {
 
-  constructor(fulfillerId, internalFulfillerId, name, email, phone, language, links) {
```

Comment:
Why does `archived` default to `undefined`? Shouldn't it be `false`?

Also... It is not set in the constructor on purpose - in which use case would people need to create a fulfiller that's already archived? IMO this only overcomplicates the constructor instead of providing sane defaults.

Reply:
undefined and false are both falsy.

Reply:
Let's do a 0 next time... :/